### PR TITLE
Updating to RTM CLI

### PR DESCRIPTION
**Pull request by joperezr**
Replacing #24433 with a PR against dev/release/2.0.0

cc: @weshaggard @eerhardt

### CODE REVIEW

**Comment by joperezr** [comment]
Looks like we don't have CI setup for PRs against this branch, and at least one I had the PR against release branch I did have a few legs failing (OSX and Tizen). Do we have a way of testing that here? Are we planning on setting up CI for this branch soon?

**Comment by joperezr** [comment]
mm that's weird I'm looking at PR #24338 which was also against this branch and that one does have CI results.

**Comment by weshaggard** [comment]
@joperezr correct there is no CI for this branch.

File: init-tools.sh [hunk 1]
```diff
@@ -46,18 +46,18 @@ if [ ! -e $__DOTNET_PATH ]; then
         case $OSName in
             Darwin)
                 OS=OSX
-                __DOTNET_PKG=dotnet-dev-osx-x64
+                __DOTNET_PKG=dotnet-sdk-${__DOTNET_TOOLS_VERSION}-osx-x64
```
Comment (eerhardt) [inline init-tools.sh]: this changed the executable bit on the .sh file.  We should fix that.

Comment (eerhardt) [reply init-tools.sh]: See the `100755 → 100644` above.

Comment (weshaggard) [reply init-tools.sh]: I already submitted the fix for this.

Comment (weshaggard) [reply init-tools.sh]: Actually I only did it in coreclr :) I will do the same for corefx.

**Pull request merged**